addons/packages/packetfence.spec

```diff
@@ -270,6 +270,8 @@ Requires: iproute, vconfig
 Requires: wmi, perl(Net::WMIClient)
 # SAML
 Requires: lasso-perl 
+# Captive Portal Dynamic Routing
+Requires: perl(Graph)
 
```

Comment:
You probably forgot to add portal_modules.conf.defaults, portal_modules.conf.example and portal_modules.conf as configuration files.